cockatrice/src/game/zones/view_zone_widget.cpp

```diff
@@ -57,13 +65,13 @@ ZoneViewWidget::ZoneViewWidget(Player *_player,
         QGraphicsProxyWidget *pileViewProxy = new QGraphicsProxyWidget;
         pileViewProxy->setWidget(&pileViewCheckBox);
         hPilebox->addItem(pileViewProxy);
-    }
 
-    if (_origZone->getIsShufflable() && (numberCards == -1)) {
```

Comment:
are you sure == -1 and <0 are the same in our code?

Reply:
does the code use negative ints beyond just -1 to encode even more info?

Reply:
I'm not sure, we'd have to check

Reply:
if it does appear that we only use -1, I think it might be best for readability if we add a const with value -1 and == to that

Reply:
seems like the constructor does get called with `-2` somewhere
https://github.com/Cockatrice/Cockatrice/blob/f2b0fa164e4ba7140f54f0633a3e27c1be8f79cc/cockatrice/src/game/game_scene.cpp#L172

Reply:
allright, important to keep track of that!